src/output/icons.rs

```diff
@@ -118,6 +118,7 @@ pub fn icon_for_file(file: &File<'_>) -> char {
             "bash_profile"  => '\u{f489}', // 
             "bashrc"        => '\u{f489}', // 
             "bat"           => '\u{f17a}', // 
+            "bats"          => '\u{f489}', // 
```

Comment:
Ah I see, the comment is wrong but the code is correct.
```suggestion
            "bats"          => '\u{f489}', // 
```